serverless/puppeteer-screenshots/src/index.js

```diff
@@ -0,0 +1,55 @@
+const setup = require('./configs/setup');
+
+exports.handler = async (event, context, callback) => {
+  // For keeping the browser launch
+  context.callbackWaitsForEmptyEventLoop = false;
+  const message = JSON.parse(event.Records[0].Sns.Message);
+  const url = message.url;
+  const key = message.key;
+  if (!url || !key) {
+    return callback('Must include url and key in the event');
+  }
+
+  const browser = await setup.getBrowser();
+  await exports.run(browser, url, key);
+  return callback(null, 'success');
+};
+
+exports.run = async (browser, url, key) => {
+  const page = await browser.newPage();
+  await page.goto(url,
+   {waitUntil: ['domcontentloaded', 'networkidle0']}
+  );
+  await page.setViewport({width: 1464, height: 764});
+  await page.screenshot({
+    path: '/tmp/screenshot.png',
+    clip: {
+      x: 64,
+      y: 64,
+      width: 1400,
+      height: 700,
+    },
+  });
+
+  const aws = require('aws-sdk');
```

Comment:
Not 100% sure but moving `require`'s up could improve performance when the function is warm.